packages/ppx_deriving_jsoo/ppx_deriving_jsoo.0.3/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+synopsis: "Ppx deriver for Js_of_ocaml"
+maintainer: "contact@origin-labs.com"
+authors: "Maxime Levillain <maxime.levillain@origin-labs.com"
+license: "LGPL-2.1-or-later"
+homepage: "https://gitlab.com/o-labs/ppx_deriving_jsoo"
+bug-reports: "https://gitlab.com/o-labs/ppx_deriving_jsoo/-/issues"
+depends: [
+  "dune" {>= "2.0"}
+  "ocaml" {>= "4.08"}
+  "ppxlib" {>= "0.18.0"}
+]
+depopts: ["ezjs_min"]
+conflicts: [
+  "ezjs_min" {< "0.2.1"}
+]
+build: [
+  ["dune" "subst"] {pinned}
```

Comment:
This is a bug in dune's opam file generation. To fix that you need to change `(lang dune 2.7)` or above or stop dune from generating the opam file.
```suggestion
  ["dune" "subst"] {dev}
```